corona-warn-app/cwa-app-android
# Navigation to test certificate from test result (EXPOSUREAPP-8688)

Navigation from negative test results with certificate to certificate details + updated screenshot. Tested on Samsung S8.

<img width="227" alt="Screenshot 2021-08-10 at 21 46 36" src="https://user-images.githubusercontent.com/64849422/128917530-cc096ab3-c963-4bd2-8bbf-eb200c6d3f81.png">

**How to test:**
1) Generate negative PCR test with dgc tc
2) Scan it
3) Wait some seconds
4) Approve/confirm test certificate (if you are using cwa cli)
5) In some seconds certificate card should appear
6) Proceed with same steps for Rapid Antigen Test

**Note to reviewer:**
I could not use same test certificate layout we used in other fragments (e.g. person details fragment) as it has different fields and margins.
